src/Http/Http.Results/src/ProblemHttpResult.cs

```diff
@@ -20,6 +20,8 @@ public sealed class ProblemHttpResult : IResult
     /// <param name="problemDetails">The <see cref="ProblemDetails"/> instance to format in the entity body.</param>
     internal ProblemHttpResult(ProblemDetails problemDetails)
     {
+        ArgumentNullException.ThrowIfNull(problemDetails);
```

Comment:
We don't usually throw this from internal methods I thought.

Reply:
Uhm! I was trying to match with:

https://github.com/dotnet/aspnetcore/blob/395eeea8a9d15b766b1bf9dd6e4655ac9b3baa17/src/Http/Http.Results/src/ValidationProblem.cs#L18

But let me remove it.

Reply:
Doh, seems I missed removing one in that commit 😵‍💫